app/src/main/java/org/jellyfin/androidtv/preference/UserPreferences.kt

```diff
@@ -180,6 +180,16 @@ class UserPreferences(context: Context) : SharedPreferenceStore(
 		 * Enable series thumbnails in home screen rows
 		 */
 		var seriesThumbnailsEnabled = Preference.boolean("pref_enable_series_thumbnails", true)
+			
+		/**
+		 * Enable subtitles background
+		 */
+		var subtitlesBackgroundEnabled = Preference.boolean("pref_enable_subtitles_background", true)
+
+		/**
+		 * Set default subtitles font size
+		 */
+		var defaultSubtitlesSize = Preference.enum("pref_subtitles_size", SubtitlesSize.SUBTITLES_SIZE_NORMAL)
```

Comment:
```suggestion
		var defaultSubtitlesSize = Preference.enum("subtitle_size", SubtitlesSize.SUBTITLES_SIZE_NORMAL)
```

I'd prefer storing an integer/double with the used size.